share/docker/casa-run/ubuntu-18.04/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:18.04
+
+# Use a minimal UTF-8-aware locale.
+ENV LANG=C.UTF-8
+
+COPY install_apt_dependencies.sh \
+     neurodebian.sources.list \
+     neurodebian-key.gpg \
+     /tmp/
+RUN /tmp/install_apt_dependencies.sh
+
+COPY install_pip_dependencies.sh /tmp/
+RUN /tmp/install_pip_dependencies.sh
+
+COPY install_compiled_dependencies.sh \
+     build_netcdf.sh \
+     build_sip_pyqt.sh \
+     /tmp/
+RUN /tmp/install_compiled_dependencies.sh
+
+COPY cleanup_build_dependencies.sh /tmp/
+RUN /tmp/cleanup_build_dependencies.sh
+
+
+RUN rm -f /tmp/neurodebian-key.gpg \
```

Comment:
It seems that this cleanup makes sense for VirtualBox too. It should go in on of the shell script then.

Reply:
It could, but I thought that it is cleaner if the process that copies the files (`Dockerfile` in my case) is also the one responsible for deleting them.